src/lib.rs

```diff
@@ -83,12 +105,15 @@ pub fn run(args: Args) {
             .expect(&format!("Unknown capture name: `{}`", capture_name))
     });
     let output_mode = get_output_mode(&args);
+    let buffer_writer = BufferWriter::stdout(ColorChoice::Never);
 
-    get_project_file_walker(&*supported_language)
+    get_project_file_walker(&*supported_language, &args.use_paths())
         .into_parallel_iterator()
         .for_each(|project_file_dir_entry| {
-            let mut printer = get_printer(output_mode);
-            let path = format_relative_path(project_file_dir_entry.path(), output_mode);
+            let printer = get_printer(&buffer_writer, output_mode);
+            let mut printer = printer.borrow_mut();
+            let path =
+                format_relative_path(project_file_dir_entry.path(), args.is_using_default_paths());
```

Comment:
Here we make the presence of leading `./` a function of whether we're in "default paths" mode or not (per `ripgrep`'s apparent behavior)